test/gtest/ucp/test_ucp_am.cc

```diff
@@ -1333,14 +1326,14 @@ class test_ucp_am_nbx_eager_data_release : public test_ucp_am_nbx {
     am_data_handler(const void *header, size_t header_length, void *data,
                     size_t length, const ucp_am_recv_param_t *rx_param)
     {
-        EXPECT_FALSE(m_am_received);
+        EXPECT_LT(m_recv_counter, m_send_counter);
```

Comment:
EXPECT_EQ(0, m_recv_counter)

Reply:
I can change it because it's non common code. Will be more clear, although the conditions has the same logic